client/app.ts

```diff
@@ -1,31 +1,37 @@
-import { sendMessage } from "./server.js";
+import { sendAction, sendChatMessage } from './server.js';
+import {
+    chatData,
+    formatMessage,
+    printMessage,
+    initializeChatListener,
+} from './chat.js';
 declare var io: any;
 
 type Canvas = {
-    canvas: HTMLCanvasElement,
-    ctx: CanvasRenderingContext2D
-}
+    canvas: HTMLCanvasElement;
```

Comment:
@bdjewkes just checking the code now. My editor must have done this when I hit auto-format. Do you know if there are supposed to be commas or semicolons there? I can change it back.

Reply:
Either are legal.  Semi-colons are more idiomatic, I think.

Reply:
If you prefer one or the other, let me know and I'll see if I can fix my formatter or switch to an editor without those plugins. I tend to hit `space-f` out of habit, and that reformats things according to whatever linter rules it has (currently [prettier](https://prettier.io/)). :/

Reply:
Actually, it looks like I can turn it off with a `.prettierignore` file instead of a `.prettierrc` file.